src/tools/ddl2cpp.cpp

```diff
@@ -141,6 +146,7 @@ class CxxModelPrinter
         output << "// SPDX-License-Identifier: Apache-2.0\n";
         output << "#pragma once\n";
         output << "#include <Lightweight/DataMapper/DataMapper.hpp>\n";
+        output << "using namespace std::string_view_literals;\n";
```

Comment:
```suggestion
        output << '\n';
        output << "using namespace std::string_view_literals;\n";
```
We should also have a separation line between pragma once and the include section, btw